config/crd/bases/psmdb.percona.com_perconaservermongodbbackups.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.1
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
please update your controller-gen. you can remove `bin/controller-gen` and run `make controller-gen`